reconcile/utils/github_api.py

```diff
@@ -38,6 +42,22 @@ def __init__(
             git_cli = Github(token, base_url=GH_BASE_URL, timeout=timeout)
         self._repo = git_cli.get_repo(repo)
 
+    def __enter__(self) -> "GithubRepositoryApi":
+        return self
+
+    def __exit__(
+        self,
+        exc_type: Optional[Type[BaseException]],
+        exc_value: Optional[BaseException],
+        traceback: Optional[TracebackType],
+    ) -> None:
+        self.cleanup()
+
+    def cleanup(self) -> None:
+        """
+        Align with GitLabApi
+        """
+
```

Comment:
So just to confirm, the GitHub API doesn't allow us to manage sessions like the GitLab api?

Reply:
No, it's no exposed to user, all managed by PyGitHub internally.